mocks/HistoryEventIterator.go

```diff
@@ -22,12 +22,16 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-// Code generated by mockery v1.0.0. DO NOT EDIT.
+// Code generated by mockery v1.0.0.
+// Modified manually for type alias to work correctly.
+// https://github.com/vektra/mockery/issues/236
+
 package mocks
 
 import (
+	"go.temporal.io/api/history/v1"
```

Comment:
It  is our convention to name proto modules *pb like `historypb`. We should stick to that convention here

Reply:
@Quinn-With-Two-Ns thanks. Iterated.